lib/modules/manager/woodpecker/index.ts

```diff
@@ -7,7 +7,7 @@ export const language: ProgrammingLanguage = 'docker';
 export { extractPackageFile };
 
 export const defaultConfig = {
-  fileMatch: ['^\\.woodpecker(?:\\/[^/]+)?\\.ya?ml$'],
+  fileMatch: ['(^|/)\\.woodpecker(?:/[^/]+)?\\.ya?ml$'],
```

Comment:
```suggestion
  fileMatch: ['^\\.woodpecker(?:/[^/]+)?\\.ya?ml$'],
```
only supported on root folder from woodpecker